### DISCUSSION TITLE
Will this feature will generate a cost?

### DISCUSSION THREAD

**Discussion by phken91** [microfeed/microfeed#74] [Q&A] [answered]
I am interested in using this platform as my personal blog. My concern is whether it will generate any costs.

**Comment by orawalters** [comment, marked as answer]
Generally that depends on the amount of traffic your blog will receive. You can use [billing notifications](https://developers.cloudflare.com/support/account-management-billing/billing-cloudflare-add-on-services/understanding-billing-for-add-on-services/#usage-based-billing-notifications) to stay aware.
